tower-test/Cargo.toml

```diff
@@ -22,9 +22,8 @@ categories = ["asynchronous", "network-programming"]
 edition = "2018"
 
 [dependencies]
-futures-util-preview = "=0.3.0-alpha.19"
-futures-executor-preview = "=0.3.0-alpha.19"
-tokio-test = "=0.2.0-alpha.6"
-tokio-sync = "=0.2.0-alpha.6"
-tower-service = { version = "=0.3.0-alpha.2", path = "../tower-service" }
+futures-util = "0.3"
+tokio = { version = "0.2", features = ["sync"]}
+tokio-test = "0.2"
+tower-service = { version = "0.3", path = "../tower-service" }
```

Comment:
I thought the idea was to only have `path` dependencies if you depend on unreleased changes?

Reply:
Not sure it works for now, if we run into issues we can remove 🤷‍♂

Reply:
I think it causes some pain with people depending on git versions of the `tower-` crates, but I forget exactly how.